src/Infusionsoft/Api/Rest/SubscriptionService.php

```diff
@@ -0,0 +1,19 @@
+<?php namespace Infusionsoft\Api\Rest;
+
+use Infusionsoft\Api\Rest\Traits\CannotDelete;
+use Infusionsoft\Api\Rest\Traits\CannotSync;
+
+class SubscriptionService extends RestModel
+{
+    use CannotSync, CannotDelete;
+
+    public $full_url = 'https://api.infusionsoft.com/crm/rest/v1//subscriptions';
+
+    public $return_key = 'subscriptions';
+
+    public function list($attributes = [])
```

Comment:
You don't need to create this function to list all as `->all()` does this for you

Reply:
Thank you. I removed the method.